src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -87,6 +87,7 @@ services:
 
   PrestaShopBundle\Form\Admin\Sell\Order\Invoices\GenerateByStatusType:
     arguments:
+      $translator: '@translator'
```

Comment:
GenerateByStatusType construct has 4 arguments and this service declaration has 3. Is it OK?

Reply:
Yes, `$locales` is filled by `bind` 
<img width="266" height="222" alt="image" src="https://github.com/user-attachments/assets/728aeb27-c1bd-45d2-95ab-a8bb8c2a803b" />